app-i18n/uchardet/uchardet-9999.ebuild

```diff
@@ -7,10 +7,7 @@ inherit cmake-utils git-r3
 
 DESCRIPTION="An encoding detector library"
 HOMEPAGE="https://www.freedesktop.org/wiki/Software/uchardet/"
-EGIT_REPO_URI=(
-	https://anongit.freedesktop.org/git/${PN}/${PN}.git
-	git://anongit.freedesktop.org/${PN}/${PN}
-)
+EGIT_REPO_URI="https://anongit.freedesktop.org/git/${PN}/${PN}.git"
```

Comment:
I understand both points of view, but would argue that the security-first view should be the default for the following reason:

1) Reliability-first users who want to allow insecure git protocols to be tried to exhaust all possible ways to download the repository (even methods which are insecure) before failing. These users will receive instant feedback that connections to remote https:// git repositories are not working, and can be reminded immediately that they _could_ change a configuration value to also attempt to use insecure protocols for git if they are willing to accept the risk.

2) Security-first users who do not want to fall back to insecure git protocols. These users have no simple way of knowing whether their security has been compromised by a MITM attacker, and therefore would need to be advised during installation/configuration of Gentoo to opt into greater security by disabling insecure git protocols. This advice/reminder would likely be missed/ignored by users, even if it was output every time an insecure git repository download occurred.

In either case, the default value doesn't really change the fact that a new configuration option in make.conf would likely need to be added to allow users to tell Gentoo package management applications what level of risk they're willing to accept with the use of insecure protocols. Some users may want to ban the use of insecure protocols altogether such that they're only used when the user has explicitly provided permission on a case-by-case basis. Some users may not ever want to use secure protocols unless they've explicitly decided otherwise on a case-by-case basis. There are also users would would prefer use of secure protocols where available, but ultimately don't care if Gentoo package management tools can automatically revert to using insecure protocols without any case-by-case permissions needing to be granted by the user.